src/template.rs

```diff
@@ -27,6 +27,80 @@ fn engine() -> liquid::Parser {
         .expect("can't fail due to no partials support")
 }
 
+pub(crate) fn conditional_engine() -> liquid::Parser {
+    liquid::ParserBuilder::with_stdlib()
+        .filter(TruthyFilterParser)
+        .filter(FalsyFilterParser)
+        .build()
+        .expect("can't fail due to no partials support")
+}
+
+#[derive(Clone, ParseFilter, FilterReflection)]
+#[filter(
+    name = "truthy",
+    description = "returns 'true' if variable is considered truthy",
+    parsed(TruthyFilter)
+)]
+pub(crate) struct TruthyFilterParser;
+
+#[derive(Debug, Default, liquid_derive::Display_filter)]
+#[name = "truthy"]
+struct TruthyFilter;
+
+impl Filter for TruthyFilter {
+    fn evaluate(
+        &self,
+        input: &dyn ValueView,
+        _runtime: &dyn Runtime,
+    ) -> Result<liquid_core::model::Value, liquid_core::error::Error> {
+        let input = input
+            .as_scalar()
+            .ok_or_else(|| liquid_core::error::Error::with_msg("String expected"))?;
+        let input = input.into_string().to_string().to_lowercase();
+
+        Ok(liquid_core::model::Value::scalar(
+            if !input.is_empty() && input != "false" && input != "0" {
```

Comment:
this should be documented in the readme before we ship this

Reply:
`README.md` didn't exactly receive any attention yesterday - as noted: this was first draft, to let you all comment on a concrete implementation of @MarcoIeni's idea (with a little extra spice!).